### DISCUSSION TITLE
moment.js startOf('week') function equivalent in arrow?

### DISCUSSION THREAD

**Discussion by nikkwong** [crsmithdev/arrow#1133] [Q&A] [answered]
Hi, I'm looking for an equivalent of this function in arrow. Thank you for any direction.

**Comment by krisfremen** [comment, marked as answer]
@nikkwong hey there!

The naming of the moment.js functions `startOf` and `endOf` in arrow are `floor` and `ceil` respectively:

```
>>> import arrow
>>> arrow.utcnow().floor('week')
<Arrow [2022-10-10T00:00:00+00:00]>
>>> arrow.utcnow().ceil('week')
<Arrow [2022-10-16T23:59:59.999999+00:00]>
```
